### PR TITLE
add style for request page and git data from forms 

### PR DESCRIPTION
**Pull request by Jamalat-shamallakh**
- add css style for request page 
- git all data from frontside and send it to back

### CODE REVIEW

File: public/css/user/request.css [hunk 1]
```diff
@@ -1,3 +1,121 @@
...
+  }
+  .main{
+    margin-top: 85px;
+  }
+  .nav--form {
+    display: flex;
+    justify-content: center
+  }
+  .nav--form-personalDetails {
+
```
Comment (InassTubail) [inline public/css/user/request.css:15]: fix indentation

**Review by InassTubail — CHANGES_REQUESTED** [review]
fix indentation

File: public/js/user/requestDom.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+const {
+  pesonalSection, requestSection,
+  personalDetails, yourRequest, fullNameSpan, phoneNumberSpan, cityNameSpan, streetNameSpan, requestSubmit, popUpSection, next,
+} = querySelectors(['pesonalSection', 'requestSection', 'personalDetails', 'yourRequest', 'fullNameSpan', 'phoneNumberSpan', 'cityNameSpan', 'streetNameSpan', 'requestSubmit', 'popUpSection', 'next'], ['.pesonal--section', '.request--section', '.personal--details', '.your--request', '.full--name-span', '.phone--number-span', '.city--name-span', '.street--name-span', '.request--submit', '.popUpBack', '.next--button']);
+// const season = document.querySelector('[name = season]');
```
Comment (InassTubail) [inline public/js/user/requestDom.js:5]: remove comments

File: public/css/user/request.css (hunk 1 above)
Comment (Jamalat-shamallakh) [reply public/css/user/request.css:15]: I did it can you make a review?

**Review by InassTubail — CHANGES_REQUESTED** [review]
`}.

.next {`
in public/css/user/donate.css line 73 
extra "."

File: public/css/user/request.css (hunk 1 above)
Comment (InassTubail) [reply public/css/user/request.css:15]: see [another requested cahnge](https://github.com/FACG6/enhance-a-smile/pull/102#pullrequestreview-217138723)

**Comment by InassTubail** [comment]
@ahmedisam99  @aminalakhsham  @AbdallahAmmar96

**Pull request closed without merging**